var/spack/repos/builtin/packages/gmp/package.py

```diff
@@ -37,6 +37,9 @@ class Gmp(AutotoolsPackage, GNUMirrorPackage):
     )
     variant("cxx", default=True, description="Enable C++ support")
 
+    # can be removed upon  >= v6.2.2
+    patch("avoid-x18.diff")
```

Comment:
```suggestion
    patch("avoid-x18.diff", when="@:6.2.1")
```
Can you add a comment pointing to the PR/patch that will be included in 6.2.2?

Reply:
Good idea! Thanks for the suggestion